core/src/commonMain/kotlin/com/ustadmobile/core/contentjob/JobConnectivityLiveData.kt

```diff
@@ -0,0 +1,37 @@
+package com.ustadmobile.core.contentjob
+
+import com.ustadmobile.core.networkmanager.ConnectivityLiveData
+import com.ustadmobile.door.DoorLiveData
+import com.ustadmobile.door.DoorMediatorLiveData
+
```

Comment:
Let's add a comment about why this class is here...

e.g.

LiveDataMediator that combines live data for the current connectivity of a job and whether or not the job can run on a metered network. This LiveData can then be used as a source to make a decision about whether or not a job needs to stop.